docker_explorer/explorer.py

```diff
@@ -202,15 +202,14 @@ def GetContainersJson(self, only_running=False):
       if container_object.mount_id:
         container_json['mount_id'] = container_object.mount_id
 
-      if container_object.mount_points:
-        container_json['mount_points'] = {}
-        for mount_point, details in container_object.mount_points.items():
+      mount_points = container_object.GetMountpoints()
+      if mount_points:
+        container_json['mount_points'] = []
+        for source, mount_point in mount_points:
```

Comment:
my understanding is that the mount_points tuple hold the src_path and dst_path of every mount point, and then you use the variable d to hold the mount point in order and  append it to container_json. Can you maybe use other variable names that are more explicit here.
ex: mount_point instead of d
src_mnt and dst_mnt instead of source and mount_point?

Reply:
done